Update react and some other dependencies

ptomasko opened the pull request:
Updated react and some other packages to be compatible with other repositories.

xurban42 reviewed (commented):
Could you also edit this package version?

ptomasko commented:
@xurban42 I cannot reply to your comment in a form of a message thread for some reason. Changed the ver to 1.2.2 , hopefully ok. Please re-check - I fixed some incompatibilities between the example and the pkg itself.
